src/types/transactions.rs

```diff
@@ -171,17 +174,23 @@ impl<P: PayloadLike> AccountTransaction<P> {
         verify_signature_transaction_sign_hash(keys, &hash, &self.signature)
     }
 }
+#[derive(Debug, Copy, Clone)]
+pub struct AddBakerKeysMarker;
+#[derive(Debug, Copy, Clone)]
+pub struct UpdateBakerKeysMarker;
```

Comment:
Replace with `pub enum AddBakerKeysMarker {} ...` and document the reason for this.

This will match the hash markers. Also derive the same things
```
#[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd)]
```